data/view_in_a_map/lab.md

```diff
@@ -21,4 +21,4 @@ In this lab you will add your own layers and some existing publicly available la
 6. Save your map. 
 
 ###Bonus
-* In the Map Viewer select `Add` > `Search for Layers` > `In: ArcGIS Online` to load other pre-existing layers into your map. HINT: Try searching for weather, rain or wind.
\ No newline at end of file
+* In the Map Viewer select `Add` > `Search for Layers` > `In: ArcGIS Online` to load other pre-existing layers into your map. HINT: Try searching for weather, rain or wind. You might try changing `In: My Organization` to `In: ArcGIS Online` to see more layers.
```

Comment:
@jgravois, I get zero results when I search for weather, rain, or wind in the efsbs org. That's why I assumed that one could find these layers outside the organization. Are there some layers that are not shared with the whole org?